builtin/providers/aws/resource_aws_db_option_group_test.go

```diff
@@ -0,0 +1,266 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSDBOptionGroup_basic(t *testing.T) {
+	var v rds.OptionGroup
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSDBOptionGroupDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccAWSDBOptionGroupBasicConfig,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAWSDBOptionGroupExists("aws_db_option_group.bar", &v),
```

Comment:
Looks like we're missing `testAccCheckAWSDBOptionGroupAttributes` - which would verify that the `rds.OptionGroup` has the attributes we set on it in the config.

Right now I could yank most of the API attributes from the code and our tests would still pass since the config values end up in the state (and the state is all `TestCheckResourceAttr` looks at).